app/models.py

```diff
@@ -2593,3 +2593,37 @@ class ServiceBroadcastProviderRestriction(db.Model):
     provider = db.Column(db.String, nullable=False)
 
     created_at = db.Column(db.DateTime, nullable=False, default=datetime.datetime.utcnow)
+
+
+class WebauthnCredential(db.Model):
+    """
+    A table that stores data for registered webauthn credentials.
+    """
+    __tablename__ = "webauthn_credential"
+
+    id = db.Column(UUID(as_uuid=True), primary_key=True, nullable=False, default=uuid.uuid4)
+
+    user_id = db.Column(UUID(as_uuid=True), db.ForeignKey('users.id'), nullable=False)
+    user = db.relationship(User, backref=db.backref("webauthn_credentials"))
+
+    name = db.Column(db.String, nullable=False)
```

Comment:
Maybe not worth it in this PR (and it's not MVP) but I wondered whether it would be worth having some kind of constraint to not allow a user to create more than one key with the same name. Just to avoid confusion - similar to how we don't let a service name give multiple API keys the same name.

Reply:
We're not too concerned about this: names are optional, as they're purely for the convenience of the user. If two keys do end up with the same name, they could use the registration date to disambiguate.